### DISCUSSION TITLE
Do you know any v3 or higher server?

### DISCUSSION THREAD

**Discussion by ErsinDemirbas** [miguelgrinberg/Flask-SocketIO#1632] [Q&A]
Hello sir, I am an jr.Android Developer and created a socket.io client side with java. So I think you know there is v1 or v2 in client side for java. My problem is that my company use v3 in server side so I have to use v2 client (highest version in java) but I cant connect to their server. So I need to a v3 or higher server side url for test my client side for check the problem is in my side or in server side. Do you know any v3 or higher server?

Note: When I switch to v1 in client I can easly connect to other v2 servers.

**Comment by miguelgrinberg** [comment]
I'm sorry but I'm not familiar with the Java version. You should ask in their board instead of this one.

**Comment by jhassell** [comment]
@ErsinDemirbas I succeeded in getting a minimal Android Application client running with the latest version of the example in Flask-SocketIO, at this date. I used the socket.io-client of io.socket:socket.io-client:2.0.0 in build.gradle dependencies. Here are some additional settings:

- android:usesCleartextTraffic="true" in AndroidManifest.xml
- Flask Socket IO, version 5.1.0
- Amazon Lightsail instance running the Flask-SocketIO example, with appropriate ports opened.

Note that the nkzawa socket.io client library for Android, com.github.nkzawa:socket.io-client:0.x.y, is not currently compatible with the latest version of Flask-SocketIO.
